docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     command: -aecore expected_mine_rate ${EPOCH_MINE_RATE:-15000}
     volumes:
       - ${PWD}/docker/epoch_node1_mean16.yaml:/home/epoch/epoch.yaml
+      - ${PWD}/docker/keys/node1:/home/epoch/node/keys
```

Comment:
How does this impact ability of user to specify mining key ? https://github.com/aeternity/epoch/blob/c4938333d80e0892cc43d265d5470fcadc1b9702/docs/docker.md#persisting-data

Reply:
This is just for the docker-compose example (I hope) not sure exactly how it relates to the instructions in docker.md?

Reply:
You are right.